app/views/admin/admin/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<%= render "homepage/header" %>
+<%= render "admintab" %>
+<div class="container">
+  <h2>WorkSpace</h2>            
+  <table class="table table-bordered">
+   
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Email</th>
+        <th>Gender</th>
+        <th>status</th>
+        <th>Manegerial Rights</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @user.each do |user|%>
+        <% if !user.admin? %>
+          <tr>
+          <td><%= user.name %></td>
+          <td><%= user.email %></td>
+          <% if user.gender == false %>
+            <td>Male</td>
+          <% else %>
+            <td>Male</td>
+          <% end %>
+          <% if user.enable? %>
```

Comment:
move this to a helper also and be creative about it, try not to repeat your code.